### DISCUSSION TITLE
Indic glyphs naming not following recommendation

### DISCUSSION THREAD

**Discussion by mitradranirban** [fontra/fontra#1633] [Q&A] [answered]
The Devanagari glyphs are named like 'ka-deva'. However this naming scheme fails fontbakery as recommendation says "A glyph name must be entirely comprised of characters from the following set: A-Z a-z 0-9 .(period) (underscore). A glyph name must not start with a digit or period. There are a few exceptions such as the special glyph ".notdef"."
So the glyphs should be named like "ka_deva" to pass the tests.

**Comment by justvanrossum** [comment, marked as answer]
You need to apply "production names" when building the font. The details for this are not yet accessible from Fontra, but fontmake's default rules work pretty well: `fontmake --production-names ...`

**Comment by mitradranirban** [comment]
Thanks for clearing. However renaming the glyphs as a_deva, aa_deva etc would have helped creating fonts through automated means like github actions in google font workflow

**Reply by justvanrossum** [reply]
Applying production names should be standard in such an environment. This naming convention comes from Glyphs app, which is also widely used in Google Fonts circles.
